rewrite-maven/src/main/java/org/openrewrite/maven/cache/RocksdbMavenPomCache.java

```diff
@@ -103,22 +105,23 @@ static synchronized RocksCache getCache(String workspace) {
     CacheResult<MavenMetadata> UNAVAILABLE_METADATA = new CacheResult<>(CacheResult.State.Unavailable, null);
     CacheResult<MavenRepository> UNAVAILABLE_REPOSITORY = new CacheResult<>(CacheResult.State.Unavailable, null);
 
-    public RocksdbMavenPomCache(@Nullable File workspace) {
+    public RocksdbMavenPomCache(@Nullable Path workspace) {
 
         assert workspace != null;
 
-        if(!workspace.exists() && !workspace.mkdirs()) {
-            throw new IllegalStateException("Unable to find or create maven pom cache at " + workspace);
-        } else if (!workspace.isDirectory()) {
+        File relativePath = new File( workspace + "/.pom");
+        if(!relativePath.exists() && !relativePath.mkdirs()) {
+            throw new IllegalStateException("Unable to find or create maven pom cache at " + relativePath);
+        } else if (!relativePath.isDirectory()) {
             throw new IllegalStateException("The maven cache workspace must be a directory");
```

Comment:
Suggest including the path in this error message. While this error is unlikely if it does come up it isn't actionable without knowing where the cache is. And let's be consistent about referring to it as the "maven pom cache"